src/ui/src/partials/request_view.html

```diff
@@ -94,7 +94,11 @@ <h1 class="page-header">
     <div class="row">
       <div class="col-md-6">
         <div style="margin-bottom: 10px;">
-          <span class="h3">Output</span>
+          <span class="h3">Output
+            <button type="button" class="btn pull-right" ng-click="download();">
+              <i class="fa fa-download" ></i> Download
```

Comment:
Let's see how this looks without the "Download" text - the download icon is pretty ubiquitous so we can probably get away with something subtler.